src/commands/up/index.ts

```diff
@@ -1,59 +1,91 @@
 import * as fs from 'fs';
 import * as path from 'path';
-import logger from '../../logger';
-import { runCommand } from '../../utils/os';
-import { DEFAULT_CONFIG_FILENAME, ImageArchitecture, UpOptions } from '../../types';
+import yaml from 'js-yaml';
+import { ConsoleParser } from '@tinystacks/ops-core';
+import { YamlConsole } from '@tinystacks/ops-model';
+import logger from '../../logger/index.js';
+import { runCommand, runCommandSync } from '../../utils/os/index.js';
+import { DEFAULT_CONFIG_FILENAME, ImageArchitecture, UpOptions } from '../../types/index.js';
+import { ChildProcess } from 'child_process';
 
-const backendSuccessIndicator = 'Running on http://localhost:8000';
-const frontendSuccessIndicator = 'ready - started server on 0.0.0.0:3000';
+const BACKEND_SUCCESS_INDICATOR = 'Running on http://localhost:8000';
+const FRONTEND_SUCCESS_INDICATOR = 'ready - started server on 0.0.0.0:3000';
+
+function installDependencies (dir: string, file: string) {
+  try {
+    const configFile = fs.readFileSync(`${dir}/${file}`);
+    const configJson = (yaml.load(configFile.toString()) as any)?.Console as YamlConsole;
+    const parsedYaml = ConsoleParser.parse(configJson);
+    const dependencyDir = `/tmp/${parsedYaml.name}`;
+    fs.rmSync(dependencyDir, { recursive: true, force: true });
+    fs.mkdirSync(dependencyDir, { recursive: true });
+    const dependencies = new Set(Object.values(parsedYaml.dependencies));
```

Comment:
Does the parser default dependencies to an empty object? If not we'll need to do that here to keep Object.values from throwing

Reply:
I think it's fine if object.values throws here, we would want the command to fail if the parser didn't work correctly